packages/node_modules/@webex/internal-plugin-devices/test/integration/spec/devices.js

```diff
@@ -30,9 +30,9 @@ describe('plugin-devices', () => {
       devices = webex.internal.devices;
     });
 
-    // This will be implemented in the same pull request that `unregister()` is
-    // implemented.
-    afterEach('attempt to unregister the device', () => {});
+    // Do nothing on error, as some methods utilize sinon for `unregister()`.
+    afterEach('attempt to unregister the device', () => devices.unregister()
+      .catch(() => {}));
```

Comment:
In some cases, when overriding `unregister()` in a test, it makes this method return a rejected promise. This will attempt to unregister and move on if it fails.

Reply:
I'd prefer to see the `afterEach` block of the test that is modifying it revert the change instead of this generic error state handling.

Reply:
Implementing this.